woocommerce/woocommerce-ios
# Update screenshots for promo v2

This PR adds the changes required for the WooCommerce Promo Screenshots v2.

**To Test**
Download the raw screenshots artifact from[ this passing test run](https://github.com/woocommerce/woocommerce-ios/actions/runs/47455595) and ensure they:
- Contain stats with proper numbers
- Contains products screenshots
